common/build.gradle

```diff
@@ -45,5 +45,5 @@ jacocoTestCoverageVerification {
 }
 check.dependsOn jacocoTestCoverageVerification
 
-lombok.config['lombok.nonNull.exceptionType'] = 'JDK'
+//lombok.config['lombok.nonNull.exceptionType'] = 'JDK'
```

Comment:
You might need to add an additional `lombok.config` file under the project root, maybe you can follow this - https://github.com/opensearch-project/ml-commons/blob/main/lombok.config

Reply:
will add, thanks